libpod/adapter/pods_remote.go

```diff
@@ -42,3 +46,48 @@ func (r *LocalRuntime) RemovePods(ctx context.Context, cli *cliconfig.PodRmValue
 	}
 	return rmPods, rmErrs
 }
+
+func (r *LocalRuntime) Inspect(nameOrID string) (*Pod, error) {
+	reply, err := iopodman.PodStateData().Call(r.Conn, nameOrID)
+	if err != nil {
+		return nil, err
+	}
+	data := libpod.PodInspect{}
+	if err := json.Unmarshal([]byte(reply), &data); err != nil {
+		return nil, err
+	}
+	pod := Pod{}
+	pod.Runtime = r
+	pod.config = data.Config
+	pod.state = data.State
+	pod.containers = data.Containers
+	return &pod, nil
+}
+
+// GetLatestPod gets the latest pod and wraps it in an adapter pod
+func (r *LocalRuntime) GetLatestPod() (*Pod, error) {
```

Comment:
If we're actually exposing `latest` remotely, there need to be some disclaimers about it not working as expected with multiple people using it at the same time, but that can happen elsewhere

Reply:
we are not exposing it

Reply:
Is it not implemented in the GetPodFromContext call for remote?

Reply:
Sure, I thought you meant GetLatestContainer as an endpoint.  I can add if you feel strongly about it

Reply:
It should not be exposed.